internal/cri/server/sandbox_run.go

```diff
@@ -455,7 +455,12 @@ func (c *criService) setupPodNetwork(ctx context.Context, sandbox *sandboxstore.
 	if netPlugin == nil {
 		return errors.New("cni config not initialized")
 	}
-
```

Comment:
this is very neat now, good change to make only the `bringUpLoopback` function platform dependant